src/index.ts

```diff
@@ -59,6 +71,13 @@ const logClientMessage = (client: Config['client']) => {
     }
 };
 
+const logMissingDependenciesWarning = (client: Config['client'], dependencies: Dependencies) => {
+    const missing = clientDependencies[client].filter(d => dependencies[d] === undefined);
```

Comment:
Might be a bit cleaner to write this with `every()`

Reply:
This filters out missing dependencies, so I can print them. `every` would just return true/false